test/logr.test.js

```diff
@@ -136,7 +136,7 @@ describe('logr', () => {
         }
       });
       log(['tag1'], 'message');
-      expect(lastMessage).to.equal('[\u001b[31mtag1\u001b[0m] message');
+      expect(lastMessage).to.equal('[\u001b[41mtag1\u001b[0m] message');
```

Comment:
I see why now.  we shouldn't change the color names. We should just change the defaults in index.js